featureless-void.cabal

```diff
@@ -23,6 +23,7 @@ library
                      Helper
                      Helper.S3
                      Helper.Time
+                     Helper.Twitter.Types
```

Comment:
After going through the rest of this, it makes more sense. Also, my suggestions is less good. What about just removing "Helper" and having Twitter we a subsection of your app?